leku/src/main/java/com/schibstedspain/leku/LocationPickerActivity.java

```diff
@@ -723,7 +722,7 @@ private void startVoiceRecognitionActivity() {
       try {
         startActivityForResult(intent, REQUEST_PLACE_PICKER);
       } catch (ActivityNotFoundException e) {
-        Log.d(LocationPickerActivity.class.getName(), e.getMessage());
+        setTracking(TrackEvents.startVoiceRecognitionActivityFailed);
```

Comment:
Just a suggestion: calling this method just `track` is... sexier! 🌶 

`setTracking` doesn't express the idea that it does something right now. But it's a totally ignorable comment :·D